src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  person,
+}) => {
+  const husbandOrWife = person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+
+  const partner = person.isMarried
+    ? husbandOrWife
+    : 'I am not married';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
Descructure the person
```suggestion
        <h2 className="Person__name">{`My name is ${name}`}</h2>
```